backend/src/src-licenses/src/main/java/com/siemens/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -411,12 +416,13 @@ private License updateLicenseFromInputLicense(License license, License inputLice
         }
         license.setText(inputLicense.getText());
         license.setFullname(inputLicense.getFullname());
-        license.setId(inputLicense.getShortname());
+        license.setId(inputLicense.isSetId() ? inputLicense.getId() : inputLicense.getShortname());
```

Comment:
True, I'm sorry. I tried it in order to see, whether this solves the problem or whether there is still another problem somewhere else. Obviously,  we committed it, though. I suggest, we leave it in here and decide what is necessary when rebasing #166.